src/SFA.DAS.LevyTransferMatching.Api/Controllers/FunctionsController.cs

```diff
@@ -52,6 +56,30 @@ public async Task<IActionResult> ApplicationApproved(ApplicationApprovedRequest
             }
         }
 
+        [Route("application-approved-receiver-notification")]
+        [HttpPost]
+        public async Task<IActionResult> ApplicationApprovedReceiverNotification(ApplicationApprovedReceiverNotificationRequest request)
+        {
+            try
+            {
+                var result = await _mediator.Send(new ReceiverApplicationApprovedEmailCommand
```

Comment:
The `result` variable is not used so need not be assigned.

Reply:
var result removed now.